smriprep/workflows/surfaces.py

```diff
@@ -532,9 +538,18 @@ def init_gifti_surface_wf(*, name="gifti_surface_wf"):
         run_without_submitting=True,
     )
     fs2gii = pe.MapNode(
-        fs.MRIsConvert(out_datatype="gii", to_scanner=True), iterfield="in_file", name="fs2gii"
+        fs.MRIsConvert(out_datatype="gii", to_scanner=True), iterfield="in_file", name="fs2gii",
     )
     fix_surfs = pe.MapNode(NormalizeSurf(), iterfield="in_file", name="fix_surfs")
+    surfmorph_list = pe.Node(
+        niu.Merge(3, ravel_inputs=True),
+        name="surfmorph_list",
+        run_without_submitting=True,
+    )
+    fscurv2funcgii = pe.MapNode(
+        MRIsConvertData(out_datatype="gii", to_scanner=True, target_surface="smoothwm"),
```

Comment:
Lingering question: Should use the `?h.white` or `?h.smoothwm` surface files?

Reply:
I don't think `to_scanner` or `target_surface` will make any difference here.